docs/README.md

```diff
@@ -34,10 +34,10 @@ We want to get this out and working so we can move on to more significant functi
 
 ## Key Goals for Next Release
 
-We are assessing the [feature roadmap](/docs/ROADMAP.md) for the next and subsequent releases.
+We are assessing the [feature roadmap](https://github.com/WorldHealthOrganization/app/blob/master/docs/ROADMAP.md) for the next and subsequent releases.
```

Comment:
Actually, does this work?
```suggestion
We are assessing the [feature roadmap](docs/ROADMAP.md) for the next and subsequent releases.
```

Reply:
No, that's what it was. The suggested link only works on GitHub, not on the [published page](https://worldhealthorganization.github.io/app/).